### ISSUE TITLE
Background Process needs update on Amazon AppStore

### ISSUE THREAD

**Issue by roborozco** [visnkmr/visnkmr#14]
The current version of the background Process app that is installed is 1.3A12i
It needs to be updated.

**Comment by B-Hookken** [comment]
Yes, Still not updated on Amazon and I cannot use Google store.  Aptoide has the app but completely different version.

**Comment by tlloydukdev** [comment]
+1

**Comment by visnkmr** [comment]
Thanks for reporting.
Sometimes appstores do take some time to roll out updates based on region, I'll look into what can be done to prevent these in the future.
In the meantime, the update is already available on github (https://github.com/visnkmr/backgroundappslist/releases/latest)

**Comment by Supernurse** [comment]
Please help, it states the current version is 1.3A12i and needs to be updated. How do I proceed please?

**Comment by visnkmr** [comment]
It should already be available from the appstore.
- If not please uninstall, reboot the device, and then reinstall the app, 
If this doesn't work you have two options,
- one, wait for the updates to be available for your region, else
- two, download the apk already available on github (https://github.com/visnkmr/backgroundappslist/releases/latest). Transfer it to fire stick using wireless file manager and install.

Assumptions:
- You know how to update a fire tv app from the appstore.
- You are using a fire tv
- You installed the app from github/amazon appstore.

If any of the assumptions are wrong, do tell me.

**Comment by Supernurse** [comment]
Hello again,

What my TV is telling me is a message saying 'sorry but we have forgotten
about the Amazon App store App' and to please remind you via this profile
to update the app. Does this make any sense? It's just the IPTV app that I
cannot log into and it's not accepting any of my log in details!

**Comment by visnkmr** [comment]
Background Apps and Process List is in no way related to any other third party app. I'm unable to estimate the cause of the issue here. The app not updating should have no relation with any other app on your TV. A screenshot may help.

**Comment by JediMike** [comment]
Still saying , "sorry, but it seems we have forgotten about the Amazon Appstore app" when you click on the "update app" button it just takes you to the main page of fire Stick or fire cube

**Comment by JediMike** [comment]
[]()![IMG_20231110_232107603.jpg](https://github.com/visnkmr/visnkmr/assets/115114095/e43dd6de-7da9-4f92-a0f2-fa245a40b382)

**Comment by visnkmr** [comment]
> ![IMG_20231110_232107603.jpg](https://user-images.githubusercontent.com/115114095/282217580-e43dd6de-7da9-4f92-a0f2-fa245a40b382.jpg)

Please refer https://github.com/visnkmr/visnkmr/issues/14#issuecomment-1799430622

**Comment by Supernurse** [comment]
It's all sorted now thanks, I simply needed to pop in a new URL in order to
access my IPTV.

Many thanks 👍

**Comment by JediMike** [comment]
Thanks for your speedy action and reply.

**Comment by AVellucci** [comment]
I've got a FireStick with version 1.3A12I installed and I have a message on my app that says "Update available for the app onGoogle Play Store"
Since FireSticks don't inherently have the Play Store I tried downloading the APK from Github and installing over the current app with ADB and I got
`adb: failed to install /Users/Admin/Desktop/app-release.apk: Failure [INSTALL_FAILED_UPDATE_INCOMPATIBLE: Package io.github.visnkmr.bapl signatures do not match previously installed version; ignoring!]`
I could uninstall the current version and then replace it with the github version but then I likely won't be able to update from the Amazon AppStore in future.
What I don't get is why it's telling me there is an update when the AppStore is still showing version 1.3A12I as the latest available version, or why it's mentioning the Google Play Store

**Comment by visnkmr** [comment]
The app mentioning Google play may be a mistake from our end. The latest version available on amazon appstore is 1.3a13i ([check here](https://www.amazon.com/Background-Apps-Process-List-Legacy/dp/B07JF3CDVZ), scroll to technical details and check version name) for some reason some fire stick users are unable to find the latest version on amazon appstore; the users unable to find the version seem to be choosen @ random from what i've seen. Installing the apk wont prevent you from being able to install amazon appstore updates since the apk is the same apk that is distributed via the amazon appstore.

**Comment by AVellucci** [comment]
weird, I definitely see that the app version is 1.3a13i on the amazon website. but no matter what I do it only downloads version 1.3a12i even though it's showing that 1.3a13i is the latest version. I've just gone and uninstalled the Amazon version and sideloaded the latest github version, not sure why Amazon is not serving me the latest version but I'll wait for a new update to see if it still fails to do any sort of updating again, thanks for the help

**Comment by visnkmr** [comment]
> weird, I definitely see that the app version is 1.3a13i on the amazon website. but no matter what I do it only downloads version 1.3a12i even though it's showing that 1.3a13i is the latest version. I've just gone and uninstalled the Amazon version and sideloaded the latest github version, not sure why Amazon is not serving me the latest version but I'll wait for a new update to see if it still fails to do any sort of updating again, thanks for the help

Yes its a very peculiar issue, reporting to amazon doesn't seem to help either. Glad you got it working.

**Comment by JLC251** [comment]
I am very confused with your reply. This issue is happening with the Firestick... that goes into a TV. 

So how do we use a GitHub link to download this update on a TV? 

Honestly, you should have a simple code to use in Downloader that we can use to update quickly and easily.  Do you have a Downloader code?

**Comment by AVellucci** [comment]
> I am very confused with your reply. This issue is happening with the Firestick... that goes into a TV.
> 
> So how do we use a GitHub link to download this update on a TV?
> 
> Honestly, you should have a simple code to use in Downloader that we can use to update quickly and easily. Do you have a Downloader code?

honestly you can make the downloader code yourself, it's a URL shortener, anyone on the internet can make their own. but just for simplicity sake here's one that takes you to the release page for backgroundappslist
